openedx/core/djangoapps/user_api/config/waffle.py

```diff
@@ -12,6 +12,7 @@
 
 # Switches
 ENABLE_MULTIPLE_USER_ENTERPRISES_FEATURE = u'enable_multiple_user_enterprises_feature'
+ENABLE_MULTIPLE_SSO_ACCOUNTS_ASSOCIATION_TO_SAML_USER = u'enable_multiple_sso_accounts_association_to_saml_user'
```

Comment:
Please see the [Toggles and Settings Doc-a-thon 2021 instructions](https://openedx.atlassian.net/wiki/spaces/AC/pages/2248409171/Toggles+and+Settings+Doc-a-thon+2021) for information about how to properly annotate this new feature toggle.
* Ignore instructions around the sign-up sheet or PR setup.  We just want this annotated so we know if this is temporary or not.
* In the instructions you'll find "For Legacy waffle classes:" and "Bonus refactoring:".  Both of these have minor refactoring instructions that apply here.  
  * The `waffle()` function below is a really bad pattern that we want to get rid of. Worst case, do not use it for the new `ENABLE_MULTIPLE_SSO_ACCOUNTS_ASSOCIATION_TO_SAML_USER` and leave the refactor of `ENABLE_MULTIPLE_USER_ENTERPRISES_FEATURE` to a different PR, but please do not reuse the pattern.  Also, if you choose to leave the `waffle()` method, please comment that any new switches should follow the new pattern.

Bonus: if you want to annotate `ENABLE_MULTIPLE_USER_ENTERPRISES_FEATURE` while you are in there, that would be great, but I know it is outside the scope of the original PR.

Thank you!

Reply:
@robrap Looks like those instructions are fairly new. Thank you for bringing this to our attention.

Reply:
1. Yes. The annotations have been around for a while, but the instructions for the refractors is very new.
2. We will soon be landing linting that will help ensure that toggles are properly annotated.
Thank you.

Reply:
@robrap I have made the changes for ENABLE_MULTIPLE_SSO_ACCOUNTS_ASSOCIATION_TO_SAML_USER switch and will create a ticket for refactoring ENABLE_MULTIPLE_USER_ENTERPRISES_FEATURE switch. Can you please take a look at the changes?

Reply:
@robrap I am merging this branch. If you think I need to make any changes related to the waffle switch, please feel free to comment. I will create another PR for that.

Reply:
@robrap @zamanafzal no worries, I changed the `LegacyWaffleSwitch` to `WaffleSwitch` in https://github.com/edx/edx-platform/pull/25935

Reply:
Thank you @zamanafzal and @nizarmah.  I'm a bit behind in reviews.  Sounds great.